src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -109,7 +113,7 @@ studio.configuration.site.preview.destroy.context.url: http://localhost:8080/api
 # Location pattern where default dependency resolver specific configuration for is stored
 studio.configuration.default.dependencyResolver.configBasePath: /configuration/dependency
 # File name where default dependency specific configuration is stored
-studio.configuration.default.dependencyResolver.configFileName: resolver-config.xml
+studio.configuration.default.dependencyResolver.configFileName: dependency/resolver-config.xml
```

Comment:
These are big changes, are you adding handlers in the upgrade manager?

Reply:
upgrade manager takes care of all config pattern changes. this one is handled by patterns above (previous comments).